src/emulator/auth/operations.ts

```diff
@@ -680,6 +681,23 @@ function createSessionCookie(
   return { sessionCookie };
 }
 
+function revokeToken(
+  state: ProjectState,
+  reqBody: Schemas["GoogleCloudIdentitytoolkitV2RevokeTokenRequest"]
+  // ctx: ExegesisContext
+): Schemas["GoogleCloudIdentitytoolkitV2RevokeTokenResponse"] {
+  assert(!state.disableAuth, "PROJECT_DISABLED");
+
+  assert(reqBody.idToken, "MISSING_LOCAL_ID");
+  assert(reqBody.providerId, "MISSING_PROVIDER_ID");
+  assert(reqBody.token, "MISSING_TOKEN");
+  assert(reqBody.tokenType, "MISSING_TOKEN_TYPE");
+
```

Comment:
I was expecting some kind of token deletion from the ProjectState - how does the revokeToken function emulate the revokeToken behavior in production?